cfpb/consumerfinance.gov
# Paying for College - Bug Fixes and Cypress Testing

This Pull Request fixes a few bugs, and also implements Cypress tests (which would have detected those bugs). It also cleans up some linting problems.

## Fixes
- URL parameters for expenses are properly put into the expenses fields in the app
- URL parameters for graduate fellowships and assistantships
- Fix URL parameter typo in "currentDebts"
- Update some poorly-named variables
- Refactor some of the asynchronous functions
- Fix lots of linter warnings

## How to test this PR
1. Run the cypress tests and make sure they pass
2. Test out some url parameters and make sure the URL values match the app values

## Checklist
